user_docs/en/userGuide.t2t

```diff
@@ -519,6 +518,20 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 | Report clipboard text | NVDA+c | Reports the Text on the clipboard if there is any. |
 %kc:endInclude
 
+++ Speech modes ++[SpeechModes]
+
+There are four speech modes available.
+* Talk: NVDA will just speak normally. This is the default speech mode.

```

Comment:
Small thing, but I don't like using "just" here.  Someone will be reading the user guide because they don't know how things work and "just", to me, implies it is something they should have already known (there are articles on this such as https://www.excitant.co.uk/just-is-a-dangerous-word/ ).  Perhaps:

Talk: The Default speech mode, NVDA speaks in reaction to actions such as moving the focus, notifications, or issuing commands.

Also, for consistency, should this group be in the same order as the command itself and the list on line 4065 (last change in this PR)

Reply:
I agree with @Qchristensen here. I had removed "just" in my original suggestion on the input help text for this, but it slipped back in to the user guide version. I figured @CyrilleB79 really wanted it there, so I didn't comment further.

```suggestion
- Talk: NVDA will speak normally in reaction to screen changes, notifications, and actions such as moving the focus, or issuing commands.
```

Reply:
@XLTechie, I had missed this from your feedback, but keeping "just" was not intended. Thanks for having insisted on this point.

Reply:
@Qchristensen wrote:
> Also, for consistency, should this group be in the same order as the command itself and the list on line 4065 (last change in this PR)

Yes for consistency, provided "Talk" is in first position since it is the normal mode.